### DISCUSSION TITLE
Temporarily hide draft comment

### DISCUSSION THREAD

**Discussion by farmio** [community/community#139256] [Mobile] [answered]
When a draft comment is swiped down, a dismissal modal is shown immediately. It is not possible anymore to temporarily hide a draft. 

Previously we could swipe down a comment draft and continue editing it later. This was very useful for having a look at other comments for reference or to copy&paste from them. 
It had even saved the draft when going to other issues / screens and returning later. 

<details>
<summary>ℹ️ Additional Debug Info</summary>
GitHub for iOS, Version 1.178.0 (141531614)
Device: iPhone 12 mini (iOS 18.0)
</details>

https://github.com/user-attachments/assets/5dfd0f41-4b0a-4da1-9ad1-150a9c87c3f8

**Comment by HenSquared** [comment, marked as answer]
Hi @farmio,

Thank you for submitting the feedback and video. We introduced this change to help users prevent from losing their entered text in long forms like PRs. 

However, I agree with you that it introduced friction in things like comments. I have an idea of a possible fix and will create an internal issue to track it.

**Reply by farmio** [reply]
Seems to be working again with 1.181.0
Thanks! 😃
